src/main/kotlin/org/koppakurhiev/janabot/services/subgroups/SubGroupsManager.kt

```diff
@@ -97,4 +112,14 @@ class SubGroupsManager {
     fun getAllGroups(): List<SubGroup> {
         return groups
     }
+
+    // TODO consider moving the result responsibility here and based on this Command will only show message
```

Comment:
Toto by som urcite nerobil v tejto classe, mozeme tu logiku nejak zmenit, urcite existuje krajsi sposob, aj ten enum nie je mozno zly napad, ale ten SubGroupManager ma proste sa starat o to aby mal spravne hodnoty ulozene/naloadovane a nic viac - single responsibility principle. Urcite by som odtialto nechcel vidiet vypisovanie messagov.

Reply:
Nechcel som tam vypisovat message - chcel som tam spravit enum resultov a to co teraz  vypisuje message nebude robit tie checky navyse len preto aby podla toho mohol spravit message, ale iba by zavolal ten Manager a bolo by tam napr.
`when(manager.addGroup(groupName)) {
    GROUP_ADDED -> MessageProvider.groupAdded(groupName)
    GROUP_ALREADY_EXISTS -> MessageProvider.groupAlreadyExists(groupName)
}`